### ISSUE TITLE
SN auth sample distribution is bad

### ISSUE THREAD

**Issue by jagerman** [graft-project/GraftNetwork#268]
The current SN auth sampling exhibits significant correlation from one block to the next because it selects 16 nodes from the previous block's sample list.

The non-independent sampling makes the selection pattern considerably more erratic over time, with a noticeable increase in the variance of the number of times a SN is selected.  This doesn't bias the results in any particular way, but the higher variance means it will take considerably longer for the distribution to approach a fair distribution.

The design document on selection simply states that it will do this, but provides no justification at all for its use.  This is puzzling to me: normally adding complexity is something that should be done for a specific reason, but no reason has been presented and the reasoning is not obvious, at least to me.

Perhaps the dev team can elaborate on why deliberately correlating the auth sample sublist across sequential block samples (and thus considerably driving up the selection frequency variance) is a desirable characteristic?

**Comment by Fez29** [comment]
I agree with the above and think justification should be presented for such additional complexities introduced into the code. And why its a desirable "feature"
